src/step-definitions/dental.steps.ts

```diff
@@ -14,3 +18,50 @@ Then(/^the main Dental title and description should be correct$/, async () => {
   await Page.validateDentalTitleImg();
 });
 
+Then(/^the title and description of What is Dental Insurance should be correct$/, async () => {
+  await Page.validateDentalOfferTitle();
+  await Page.validateDentalOfferrDescription();
+});
+
+Then(/^who typically needs it most\? title, pictures and descriptions for Dental should be displayed$/, async () => {
+ await Page.validateDentalFamiliesTitle();
+ await Page.validateDentalFamiliesDescription();
+ await Page.validateDentalFamiliesTitleImg();
+});
+
+Then(/^find your product button for Dental\? is working$/, async () => {
+  await Page.validateDentalButtonFindYourProductElem();
+  await Page.validateDentalButtonFindYourProductTextElem();
+  await Page.validateDentalButtonIsWorking();
+});
+
+Then(/^the Quizzard page is opened$/, async () => {
+  await _Quizzard.validateWizarCardTitle();
+  await _Quizzard.validateUrl();
+});
+
+Then(/^validate back button label and click it$/, async () => {
+  await _Quizzard.validateQuizzardBackButtonText();
```

Comment:
Instead of a method it could be just an assert here:
await expectText(this.quizzardBackButtonText, Data.quizzardButtonText);